### DISCUSSION TITLE
Datenfelder Status als Icon anzeigen

### DISCUSSION THREAD

**Discussion by gyroslav** [Zefau/ioBroker.jarvis#681] [Ideas / Feature Requests]
Hallo zusammen, ich bin neu in Github und bei Jarvis. 
Ich möchte eigentlich nur eine einfache Anleitung, wie ich ein Icon im Statusfeld anstelle des Datenwertes darstellen lassen kann. 

Situation: SMA WR, Modbus Feld gibt aus: 
35 = Fehler 
303 = Aus 
307 = OK 
455 = Warnung 

Wie ich anstelle von "307" ein "OK" anzeigen lasse, wird an jeder zweiten Stelle beschrieben. 
Aber wie ich da das "check-circle" - Icon rein kriege, gibt es nicht ein Beispiel zu? Bin ich Blind?

In Issues gibt es einen recht jungen Verweis auf ein Forum mit annährend 4000 Seiten... ich werde daraus nicht schlau. 

Danke!

**Comment by kailausberg** [comment]
Genau dieses Feature geht mit der 2.2 nicht (mehr) - aber du hast ja bereits für das feature gevotet.
Mit  der 2.1 funktionierte das wie folgt als "Anzeigevariante":
`
{
  "307": "<span class='check-circle' style='font-size:26px' title='OK'></span>",
  "455": "<span class='mdi-alarm-light' style='font-size:26px' title='Warnung!'></span>",
  .......
}
`

**Comment by Zefau** [comment]
Kommt mit v3 zurück, siehe https://github.com/Zefau/ioBroker.jarvis/issues/680
